tracing/src/main/java/org/axonframework/extensions/tracing/SpanUtils.java

```diff
@@ -39,14 +39,15 @@
     private static final String TAG_AXON_PAYLOAD_TYPE = "axon.message.payload-type";
     private static final String TAG_AXON_MESSAGE_NAME = "axon.message.message-name";
 
-
     /**
      * Registers message-specific tags to the given {@code spanBuilder} based on the given {@code message}.
      *
      * @param spanBuilder the Span Builder to register the tags with
      * @param message     the message to retrieve details from
      * @return a builder with tags attached
+     * @deprecated
```

Comment:
Isn't a reason desirable for this `@deprecated` ?

Reply:
Nope, you are right, that should be there. Sharp!